docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -167,6 +167,8 @@ NOTE: {endpoint-sec} has a different output matrix.
 |{n}
 |===
 
+Currently, {agent}s managed by Fleet can only output to the same {es} cluster where {fleet} is running. Support for outputting to remote {es} clusters is under consideration for a future release.
+
```

Comment:
@mostlyjason if we don't have a public issue for it, then please create and link it here. It will be a great way to collect inputs from users on its priority

Reply:
done